### PR TITLE
add welcome view when actuator is absent (dedicated view)

### PR DESCRIPTION
**Pull request by Eskibear**
Signed-off-by: Yan Zhang <yanzh@microsoft.com>

### CODE REVIEW

**Comment by Eskibear** [comment]
If no actuator found on current running app, it shows below view

image

**Comment by Eskibear** [comment]
Feedbacks from peers:

- In notification, provide a shortcut to "save and re-run".
- Consider when to re-present the guide after "I Understand" is clicked.

**Comment by Eskibear** [comment]
https://user-images.githubusercontent.com/2351748/186613561-1d16c092-1375-42d2-bc49-f900a82a5e15.mp4

File: package.json [hunk 1]
```diff
@@ -60,19 +60,33 @@
...
-          "when": "java:serverMode",
+          "when": "java:serverMode && !spring:actuatorRequired",
           "contextualTitle": "Spring",
           "icon": "resources/logo.png"
         },
         {
           "id": "spring.mappings",
           "name": "Endpoint Mappings",
-          "when": "java:serverMode",
+          "when": "java:serverMode && !spring:actuatorRequired",
```
Comment (jdneo) [inline package.json:70]: What's the purpose of checking `java:serverMode`?

Comment (Eskibear) [reply package.json:70]: hide for non-java workspaces

**Review by jdneo — APPROVED** [review]

**Pull request merged**